src/pages/research.astro

```diff
@@ -0,0 +1,1131 @@
+                            </span>
+                            <svg
+                              xmlns="http://www.w3.org/2000/svg"
+                              class="h-5 w-5"
+                              fill="none"
+                              viewBox="0 0 24 24"
+                              stroke="currentColor"
+                              stroke-width="2"
+                            >
+                              {/* source https://heroicons.com v1.0.6, MIT licensed */}
+                              <path
+                                stroke-linecap="round"
+                                stroke-linejoin="round"
+                                d="M10 6H6a2 2 0 00-2 2v10a2 2 0 002 2h10a2 2 0 002-2v-4M14 4h6m0 0v6m0-6L10 14"
+                              />
+                            </svg>
+                          </a>
+                        )}
+                        {paper.abstract && (
+                          <label
+                            for={getToggleAbstractId(paper)}
+                            class="btn-action abstract-toggle"
+                            aria-label="Toggle abstract"
+                            aria-expanded="false"
+                          >
+                            <span class="ml-2 text-sm font-bold text-gray-500 move-button">
+                              ABSTRACT
+                            </span>
+                            <svg
+                              xmlns="http://www.w3.org/2000/svg"
+                              fill="none"
+                              viewBox="0 0 24 24"
+                              class="h-6 w-6 transform transition-transform duration-200"
+                              stroke="currentColor"
+                              stroke-width="2"
+                            >
+                              {/* source https://heroicons.com v1.0.6, MIT licensed */}
+                              <path
+                                stroke-linecap="round"
+                                stroke-linejoin="round"
+                                d="M19 9l-7 7-7-7"
+                              />
+                            </svg>
+                          </label>
+                        )}
+                      </div>
+                    </div>
+                    <div class="abstract-content mt-4 text-sm text-gray-700">
+                      {paper.abstract &&
+                        paper.abstract
+                          .split("\n\n")
+                          .map((paragraph, i) => (
+                            <p class={i > 0 ? "mt-4" : ""}>{paragraph}</p>
+                          ))}
+                    </div>
+                  </article>
+                ))}
+              </div>
+            </section>
+          ))
+        }
+      </div>
+    </Container>
+  </div>
+</Layout>
+
+<style>
+  .btn-action {
+    @apply p-2 text-gray-400 hover:text-gray-600 rounded-full hover:bg-gray-100 transition;
```

Comment:
```suggestion
    @apply p-2 text-gray-500 hover:text-gray-600 rounded-full hover:bg-gray-100 transition;
```

Reply:
Thanks. 3c8f22f13578fb6d405576a918b54b57c7259b46 should fix this. To me it looked what I removed there (my earlier change) did not really have a visible impact.